### ISSUE TITLE
[Feature Request] dump snapshot after proposals

### ISSUE THREAD

**Issue by Lee-W** [pycontw/pycon.tw#1063]
## Is your feature request related to a problem?
nope

## Describe the solution you'd like

We're now migrating pycon.tw to python.tw. For reducing the risk, it better for us to dump snapshot to local after CfP

**Comment by mattwang44** [comment]
Do you expect any code change for this? or just dump the db data using the regular way?

**Comment by tai271828** [comment]
> Do you expect any code change for this? or just dump the db data using the regular way?

No. Not expect any code change. We are not sure if the migration of the domain[1] from `pycon.tw` to `python.tw` will impact GCP project or not. We suppose there will be no impact. Backup the data is simply for spare in case of anything that we did not think of.

[1] Precisely speaking, we are going to sunset the legacy G-suite binding the `pycon.tw` domain and launch another Google workspace binding with the `python.tw` domain.

**Comment by mattwang44** [comment]
FYI I have already dumped the DB and have it in my machine. You folks can close this issue whenever you think the migration is done and confirmed it's safe.
